tests/sentinel/tests/12-master-reboot.tcl

```diff
@@ -47,7 +47,7 @@ test "Master reboot in very short time" {
     R $master_id config rewrite
 
     foreach_sentinel_id id {
-        S $id SENTINEL SET mymaster master-reboot-down-after-period 5000
```

Comment:
i think we can keep a dummy one set/get to ensure the old name is working?

Reply:
@enjoy-binbin I just address your comment, could you please take a look? Thanks

Reply:
I mean, add something like this, the statement will use the old name to do the set and get, it can make sure we did not break the old name usage.
```
S 0 SENTINEL SET mymaster primary-reboot-down-after-period 5000
S 0 SENTINEL GET mymaster primary-reboot-down-after-period
```

Reply:
There is no `SENTINEL GET` command? Only `SENTINEL CONFIG GET` but that's not the same.

Reply:
ohh right, I thought they were a pair, may bad, sorry for that.